examples/complete/versions.tf

```
@@ -2,9 +2,9 @@ terraform {
   required_version = ">= 1.0"
 
   required_providers {
-    local = {
-      source  = "hashicorp/local"
-      version = "2.1.0"
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
```

We do not put upper limits on providers so that when new major versions come out, we automatically get to see if they break something. 

```suggestion
      version = ">= 5.0"
```